src/scripts/onerepStatsAlert.js

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Sentry from "@sentry/nextjs";
+
+const SENTRY_SLUG = "cron-onerep-stats-alerts";
+
+const MAX_MANUAL_SCANS = parseInt(process.env.MAX_MANUAL_SCANS) || 0;
+const MAX_INITIAL_SCANS = parseInt(process.env.MAX_INITIAL_SCANS) || 0;
+const MAX_PROFILES_ACTIVATED =
+  parseInt(process.env.MAX_PROFILES_ACTIVATED) || 0;
+const MAX_PROFILES_CREATED = parseInt(process.env.MAX_PROFILES_CREATED) || 0;
+
+Sentry.init({
+  environment: process.env.APP_ENV,
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
+});
+
+const checkInId = Sentry.captureCheckIn({
+  monitorSlug: SENTRY_SLUG,
+  status: "in_progress",
+});
+
+/**
+ * Fetch the latest usage statistics from OneRep's API.
+ *
+ * @see https://docs.onerep.com/#tag/Statistics
+ */
+export async function checkStats() {
+  const scans = await (await onerepFetch("/stats/scans")).json();
+  const profiles = await (await onerepFetch("/stats/profiles")).json();
```

Comment:
Possibly unpopular opinion and refactor, but it looks like `onerepFetch()` always expects/returns JSON. Should `onerepFetch()` just automatically return the `.json()` payload to avoid these parenthetical `await`s?
Or possibly a bad idea since some methods might be using `res.ok` or something to see if requests were successful.

Reply:
Yes this is a good point... I think a good refactoring would be maybe to make the `onerepFetch` function always check for `res.ok`, and make it `throw` if not.

Reply:
Is there a ticket for this refactor or doing it in this PR?

Reply:
I think it's too much for this PR, I've filed a ticket: https://mozilla-hub.atlassian.net/browse/MNTOR-2686